### PR TITLE
Add tests using Sass modules

### PR DESCRIPTION
**Pull request by kaelig**

### CODE REVIEW

File: test/test-modules.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+@use '../mq' as * with (
```
Comment (kaelig) [inline test/test-modules.scss]: I was expecting the `mq()` mixin to become part of the global scope, meaning it'd be accessible to the [`after-import` file](https://github.com/sass-mq/sass-mq/blob/master/test/_after-import.scss).

File: test/test-modules.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        // Tweakpoints
+        mobileLandscape: 480px
+    ),
+
+    /* Show active breakpoint in top right corner of viewport */
+    $mq-show-breakpoints: (mobile, mobileLandscape, tablet, desktop, widescreen, tvscreen)
+);
+
+
+@import 'after-import';
```
Comment (kaelig) [inline test/test-modules.scss]: I've attempted to make `mq()` accessible to this file from here, but couldn't figure it out.

@nex3 I might be trying to do something that isn't supported as-is in a world with Sass modules. If you have time, could we talk about this use-case so I can make sense of it all? Eventually, I'd like to write a blog post about library best practices, but first I need to really understand what's doable and what isn't 😓.

File: test/test-modules.scss (hunk 1 above)
Comment (nex3) [reply test/test-modules.scss]: The `@use` rule is *always* scoped to the current file, even with `as *`, and even when combined with `@import`. The goal is to ensure that each file explicitly refers to every member it uses.

File: test/test-modules.scss (hunk 2 above)
Comment (mirisuzanne) [reply test/test-modules.scss]: @kaelig i highly recommend this pattern:

```scss
// _mq-config.scss
@use '../mq' as * with (
    $mq-breakpoints: (
        // Base device classes
        mobile:     320px,
        tablet:     640px,
        desktop:    1024px,
        widescreen: 1200px,
        tvscreen:   1920px,

        // Tweakpoints
        mobileLandscape: 480px
    ),

    /* Show active breakpoint in top right corner of viewport */
    $mq-show-breakpoints: (mobile, mobileLandscape, tablet, desktop, widescreen, tvscreen)
);

@forward '../mq';
```

and then never import `../mq`directly - always `@use` the `_mq-config` file. You can use it inside `after-import`  if you need configured-mq there.

**Pull request closed without merging**